.actions/assistant.py

```diff
@@ -65,7 +68,12 @@ def _replace_imports(lines: List[str], mapping: List[Tuple[str, str]]) -> List[s
 def copy_replace_imports(
     source_dir: str, source_imports: List[str], target_imports: List[str], target_dir: Optional[str] = None
 ) -> None:
-    print(f"Replacing imports: {locals()}")
+    """Copy package content with import adjustments.
+
+    >>> _ = copy_replace_imports(os.path.join(
+    ...     _PATH_ROOT, "src"), ["lightning_app"], ["lightning.app"], os.path.join(_PATH_ROOT, "src", "lightning"))
```

Comment:
This doctest doesn't do anything. It will also create changes in the local directory. I suggest removing it. It's widely used in our CI

Reply:
Made the change to resolve and get the fix merged